refinery/core/models.py

```diff
@@ -2340,6 +2343,13 @@ def get_aggregate_sum(field_name):
         ]
 
 
+class SiteVideo(models.Model):
+    caption = models.TextField(blank=True)
+    site_profile = models.ForeignKey(SiteProfile)
+    source = models.CharField(max_length=100, blank=True)
```

Comment:
One thing that could be useful for the `source` field is the use of the `choices` arg.

```
class SiteVideo(models.Model):
    YOUTUBE = 'YT'

    VIDEO_SOURCE_CHOICES = (
        (YOUTUBE, 'Youtube'),
        ...
    )
    ...
    source = models.CharField(choices=VIDEO_SOURCE_CHOICES, max_length=100, blank=True)
```

Even if we don't envision videos coming from too many different sources, this would still solve the problem of potentially having many differing entries like: `youtube`, `Youtube`, `YouTube` etc.